FrameRate/FrameRate.cpp

```
@@ -72,6 +72,16 @@ namespace WPEFramework
 
             SYSLOG(Logging::Startup, (_T("FrameRate::Initialize: PID=%u"), getpid()));
 
+            try
+            {
+                device::Manager::Initialize();
+                LOGINFO("device::Manager::Initialize success");
+            }
+            catch(...)
+            {
+                LOGINFO("device::Manager::Initialize failed");
```

should we return an error (non empty string) ? Also we have a standard macro to print exception errors.

---

Added in latest commit

---

exception has been added